### ISSUE TITLE
Shotguns fix

### ISSUE THREAD

**Issue by EmSockz** [WeaponMechanics/MechanicsMain#229]
### Description

Add a different application of shotgun spread to each bullet when fired.  The shotgun fires like it's a machine gun and there's no fixing it.  (Perhaps I did not see something, but even with the standard setting it is so)

### Configuration

```yaml
My_Feature:
  Feature_Option: <true/false>
  Other_Option: <amount>
  etc.
```

### Payment (optional)

- [ ] I'm willing to pay for this feature

**Comment by DeeCaaD** [comment]
Use discord for support.

All projectiles fire have different spread and you can use things such as delay between shots to prevent shotgun spam

**Comment by CJCrafter** [comment]
^ that was meant to fix #299
